src/main/java/uk/gov/hmcts/reform/sscs/service/AirLookupService.java

```diff
@@ -39,7 +40,7 @@ public String lookupRegionalCentre(String postcode) {
     }
 
     private static AirlookupBenefitToVenue DEFAULT_VENUE = AirlookupBenefitToVenue.builder().pipVenue("Birmingham").esaOrUcVenue("Birmingham").build();
-    private static final Map<Benefit, String> BENEFIT_CODE_VENUE = Map.of(PIP, "pip", DLA, "pip", CARERS_ALLOWANCE,"pip");
+    private static final List<Benefit> PIP_BENEFIT_CODE_VENUE = List.of(PIP, DLA, CARERS_ALLOWANCE);
```

Comment:
Why is this PIP specific?

Reply:
We have logic in lookupAirVenueNameByPostCode to check if these list of benefit type then use the pip venue otherwise EsaorUC venue